app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,97 +1,143 @@
 import React, { useState, useEffect } from "react";
 
+import { Search, Settings, Plus, BurgerMenu } from "@bigbinary/neeto-icons";
+import { Typography, Input, Button } from "@bigbinary/neetoui/v2";
+import { MenuBar, Header } from "@bigbinary/neetoui/v2/layouts";
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui";
-import { Header, SubHeader } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
+import ListNote from "./ListNote";
+import { notes } from "../../../data/notes";
```

Comment:
Can we keep this constant data inside a `constants.js` file inside the scope of `Notes` directory?